rest-api/src/main/java/gov/cms/qpp/conversion/api/services/FileRetrievalService.java

```diff
@@ -0,0 +1,8 @@
+package gov.cms.qpp.conversion.api.services;
+
+import java.io.InputStream;
+import java.util.concurrent.CompletableFuture;
+
+public interface FileRetrievalService {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Document this public interface by adding an explicit description. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUndocumentedApi)